src/applications/personalization/dashboard-2/components/health-care/Prescriptions.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import HealthCareCard from './HealthCareCard';
+import { recordDashboardClick } from 'applications/personalization/dashboard/helpers';
+
+import { mhvUrl } from '~/platform/site-wide/mhv/utilities';
+
+export const Prescriptions = ({ authenticatedWithSSOe, prescriptions }) => {
+  let filteredPrescriptions;
+  let latestPrescription;
+  let nrRefillsInProgress;
+
+  if (prescriptions?.length) {
+    const thirtyDaysAgo = moment()
+      .endOf('day')
+      .subtract(30, 'days');
+    const statuses = new Set(['refillinprocess', 'submitted']);
+
+    filteredPrescriptions = prescriptions
+      .filter(p => statuses.has(p.attributes?.refillStatus))
+      .filter(
+        p =>
+          moment(p.attributes?.refillSubmitDate).isAfter(thirtyDaysAgo) ||
+          moment(p.attributes?.refillDate).isAfter(thirtyDaysAgo),
+      );
+
+    // GET LATEST DATE TO MAKE SURE LATEST PRESCRIPTION IS CORRECT
+    // This SHOULD be OK since we're filtering in loadPrescriptions
+    latestPrescription = filteredPrescriptions[0];
+    nrRefillsInProgress = filteredPrescriptions?.length;
```

Comment:
nit: not a fan of `nr` 😆  I mean unless that's some well-established convention I don't know about!